submissions/Olumide Jokotade/HTML-CSS/Frere Wears Homepage.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
```

Comment:
Remove spaces in filename

Reply:
Thank you, I have just done that. I have also only just realized that i put the wrong html file in the folder. I have replaced it with the correct one.

Reply:
I am sorry about the deletion. I thought the purpose was served since i had to do that because of an earlier error.